### DISCUSSION TITLE
How is the Glance company changing the way we consume digital content?

### DISCUSSION THREAD

**Discussion by viralsweet** [community/community#141657] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I was wondering how Glance is changing the way we consume digital content on our phones. Like, what new things they doing that makes content more easy to see or use on our lock screens? Does anyone have any thoughts or noticed anything different when using Glance compared to other apps

**Comment by LiteBrite82** [comment]
Hi @viralsweet ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
